tests/integration/helpers/factories/db/invenio_records.py

```diff
@@ -55,7 +55,7 @@ class TestRecordMetadata(TestBaseModel):
     }
 
     @classmethod
-    def create_from_kwargs(cls, index=True, has_pid=True, **kwargs):
+    def create_from_kwargs(cls, index=None, pid_type=None, **kwargs):
```

Comment:
An index is a data structure. This variable does not hold an index, but the name of an index. Thus `index_name` or something similar is a more appropriate naming.

Reply:
Also if index is a string, default it to `''`